### PR TITLE
Update changelog for 2.3.0

### PR DESCRIPTION
**Pull request by ginglis13**
**Issue number:**

N / A

**Description of changes:**

Updates the CHANGELOG for the upcoming 2.2.1 release, along with bumping the version for the release.

**Testing done:**

N / A

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
+# v2.2.1 (2024-07-24)
+
+## OS Changes
+
+* Update containerd to 1.7.2 ([#40])
```
Comment (vigh-m) [inline CHANGELOG.md]:
```suggestion
* Update containerd to 1.7.20 ([#40])
```

**Review by vigh-m — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ginglis13) [reply CHANGELOG.md]: Thanks - looking again I see I've missed the runc bump from that PR too, I'll push an udpate

**Comment by ginglis13** [comment]
^ add runc update, correct containerd version

**Comment by ginglis13** [comment]
^ correct the containerd version in BRSA

**Review by vigh-m — APPROVED** [review]

**Review by ytsssun — APPROVED** [review]

**Comment by ginglis13** [comment]
We'll target including #42 and bump the version to 2.3.0!

**Comment by ginglis13** [comment]
^ include #42, so release as 2.3.0

**Comment by ytsssun** [comment]
> ^ include #42, so release as 2.3.0

Nit: I know this is from your local branch, but would it be worth to cut a new PR from a new branch `ginglis13:changelog-2.3.0`?

**Review by vigh-m — APPROVED** [review]

**Comment by ginglis13** [comment]
> Nit: I know this is from your local branch, but would it be worth to cut a new PR from a new branch ginglis13:changelog-2.3.0?

That would be reflected in the default merge commit message, so I see your point. I'll opt to make sure the merge commit reflects 2.3.0, since I plan to delete my source branch after merge anyways.

**Review by koooosh — APPROVED** [review]

**Pull request merged**